.github/workflows/ci.yml

```diff
@@ -72,3 +71,20 @@ jobs:
         if: always()
         run: |
           util/check_local_modifications.sh
+  integration-tests:
+    name: Integration tests
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip --break-system-packages
+          python3 -m pip install -r requirements_dev.txt --break-system-packages
```

Comment:
```suggestion
          python3 -m pip install --upgrade pip
          python3 -m pip install -r requirements_dev.txt
```

Reply:
since we are installing python explicitly via GitHub action there is no need for this argument to be present